keylime/tenant.py

```diff
@@ -582,6 +585,39 @@ def do_cvadd(self):
                 response.text,
             )
             sys.exit()
+        else:
+            numtries = 0
+            added = False
+
+            while not added:
+                reponse_json = self.do_cvstatus()
+                if reponse_json["code"] != 200:
+                    numtries += 1
+                    if numtries >= self.maxr:
+                        logger.error(
+                            'Agent ID %s status is still not "added" on Verifier\'s database after %d tries',
```

Comment:
I still think this isn't direct enough. Maybe something like "Agent ID %s still not added to Verifier after %d tries", then repeat for the rest. We don't need to tell the user we keep track of it with a specific status value or how we track that internally, just the logical action of was it added or not.

Reply:
Fixed now